NEWS.md

```diff
@@ -2,8 +2,9 @@
 
 Updated package for:
 
-- Use in DataCamp's [Modeling with Data in the Tidyverse](https://www.datacamp.com/courses/modeling-with-data-in-the-tidyverse), in particular added `evals` and `house_prices` datasets and updated `get_regression_table()` and `get_regression_points()` functions.
-- v0.4.0 of [ModernDive](https://moderndive.github.io/moderndive_book/) textbook
+- Use in DataCamp's [Modeling with Data in the Tidyverse](https://www.datacamp.com/courses/modeling-with-data-in-the-tidyverse), in particular added `evals` and `house_prices` datasets and updated `get_regression_table()` and `get_regression_points()` functions. 
+- v0.4.0 of [ModernDive](https://moderndive.github.io/moderndive_book/) textbook by including
```

Comment:
Add mention of adding `evals` and `house_prices` datasets in the v0.4.0 of ModernDive as well, since those datasets get used there as well.

Reply:
testing slack app notifications